config.toml

```diff
@@ -27,12 +28,12 @@ staticDir = ["static"]
     url = "/"
     weight = 1
 [[menu.main]]
-    name = "Documents"
+    name = "Documentation"
     url = "/docs/"
     weight = 2
 [[menu.main]]
-    name = "Help"
-    url = "/help/"
+    name = "Get Involved"
```

Comment:
Help is a better name here as it's not necessarily about the community.

Reply:
If someone wants help, don't they want to get in touch with the community?

If someone wants to self-serve, then they'll look at documents, right?